iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/writer/HiveIcebergDeleteWriter.java

```diff
@@ -58,12 +59,12 @@ public void write(Writable row) throws IOException {
     Record rec = ((Container<Record>) row).get();
     PositionDelete<Record> positionDelete = IcebergAcidUtil.getPositionDelete(rec, rowDataTemplate);
     int specId = IcebergAcidUtil.parseSpecId(rec);
-    Record rowData = positionDelete.row();
+    PartitionKey partitionKey = IcebergAcidUtil.parsePartitionKey(rec);
```

Comment:
originally data was evaluated from the position delete row data. would it still be the same if merge functionality is disabled?

Reply:
Added support for this. Done.